src/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -11,37 +11,44 @@
 import java.awt.geom.Point2D;
 import java.awt.image.BufferedImage;
 import java.util.List;
+import java.util.function.Function;
 
 import org.apache.commons.math3.analysis.interpolation.SplineInterpolator;
 import org.apache.commons.math3.analysis.polynomials.PolynomialSplineFunction;
 
 import games.strategy.engine.data.Route;
 import games.strategy.engine.data.Territory;
-import games.strategy.util.Tuple;
 
 /**
  * Draws a route on a map.
  */
 public class MapRouteDrawer {
 
   private static final SplineInterpolator splineInterpolator = new SplineInterpolator();
+  /**
+   * This value influences the "resolution" of the Path.
+   * Too low values make the Path look edgy, too high values will cause lag and rendering errors
+   * because the distance between the drawing segments is shorter than 2 pixels 
+   */
+  public static final double DETAIL_LEVEL = 1.0D;
```

Comment:
The value "1.0D" can simply be 1.0
Floats and longs are the only types you really need to annotation with "f" or "L"